addon-test-support/@ember/test-helpers/dom/focus.ts

```diff
@@ -18,7 +18,15 @@ registerHook('focus', 'start', (target: Target) => {
 */
 export function __focus__(element: HTMLElement | Element | Document | SVGElement): void {
   if (!isFocusable(element)) {
-    throw new Error(`${element} is not focusable`);
+    if (
+      document.activeElement &&
+      document.activeElement !== element &&
+      isFocusable(document.activeElement)
+    ) {
+      __blur__(document.activeElement, null);
+    }
+
+    return;
```

Comment:
This seems like we are basically duplicating the code just below (line 36), but after a closer inspection I can see that it is quite different (e.g. we are only doing it if the target is not focusable, and we do it regardless of `document.hasFocus()`).

Can you add an inline comment to explain why this exists?

Reply:
Done